// Copyright 2017 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package modemfwd;

message HelperEntry {
  string filename = 1;
  repeated string device_id = 2;
}

message HelperManifest {
  repeated HelperEntry helper = 1;
}
